onnxscript/rewriter/ort_fusions/attention_basic_test.py

```diff
@@ -0,0 +1,120 @@
+# Copyright (c) Microsoft Corporation.
```

Comment:
To clarify PR #2142  further: this type of test is more for debugging (the mathematical rewrite rule), and I was suggesting it in case the relative behavior of the two ops was unclear. 

For the rewrite rule implementation, you should have a test case where the rewrite rule is applied. Eg., you can apply it to the "expanded model" defined below, and verify that the rewritten model has the expected attention op, and that it behaves equivalent to the original model.